### PR TITLE
ci: add tagger workflow

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: .github/workflows/tagger.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Check current version
+        id: check_version
+        run: echo ::set-output name=VERSION::$(node -p "require('./package.json').version")
```
Comment (shigma) [inline .github/workflows/tagger.yml:20]: `set-output` is deprecated?

```sh
echo "VERSION=$(xxxxx)" >> $GITHUB_OUTPUT
```

**Review by shigma — CHANGES_REQUESTED** [review]

File: .github/workflows/tagger.yml [hunk 2]
```diff
@@ -17,32 +17,27 @@ jobs:
 
       - name: Check current version
         id: check_version
-        run: echo ::set-output name=VERSION::$(node -p "require('./package.json').version")
+        run: echo "VERSION=$(node -p "require('./packages/core/package.json').version")" >> $GITHUB_OUTPUT
```
Comment (shigma) [inline .github/workflows/tagger.yml:20]: It seems that `"` is not escaped properly.

Comment (MaikoTan) [reply .github/workflows/tagger.yml:20]: That is not required to escape quotes in bash, I don't know why but it seems working.
image

Comment (shigma) [reply .github/workflows/tagger.yml:20]: Oh that's really amazing.

**Review by shigma — APPROVED** [review]

**Pull request merged**